### ISSUE TITLE
`activateSystemCursor` should include a view ID

### ISSUE THREAD

**Issue by mdebbar** [flutter/flutter#140226]
At least on the web, the cursor can be set at whatever scope we choose (e.g. entire window, flutter view element, etc). So in order for us to correctly scope the cursor, we would need the `viewId` information within the `activateSystemCursor` message.

**Comment by goderbauer** [comment]
cc @dkwingsmt 

I suspect that this is going to be difficult because as far as I remember the mouse system doesn't associate the cursor with a view. It sees the cursor as "global".

**Comment by dkwingsmt** [comment]
Yeah. Normally such cursor changes are associated with widgets, which can be tracked down to a view. However, the cursor API itself is not built with this layer in mind, which allows arbitrary cursor changes unassociated with views. Requiring a view ID in `activateSystemCursor` might sound confusing.

Can we just set all views to use this cursor? It might impact the performance but I think the browser must have some kind of optimization.

Another way is to have the Web manually track the enter/leave events for the view that the cursor is currently in.

**Comment by dkwingsmt** [comment]
Now that I look at it, I think we should apply the cursor to all Flutter views, and therefore `activateSystemCursor` should not include a view ID. As far as I see, Web is designed to support only a single mouse cursor.

As for @holzgeist 's issue, I'm not familiar with the PIP feature, but your description sounds like a bug to me. We should apply the cursor to the correct div. I'll appreciate it if you can file an issue with a complete reproduction. Thank you.
